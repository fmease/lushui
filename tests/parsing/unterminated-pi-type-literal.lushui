;; TEST fail check

data X (A: Type): (m: M) =
