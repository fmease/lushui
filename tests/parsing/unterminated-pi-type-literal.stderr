error[E010]: found `=`, but expected `->`
 --> ${DIRECTORY}/parsing/unterminated-pi-type-literal.lushui:3:19
  |
3 | data X (A: Type): (m: M) =
  |                   ------ the start of a pi type literal
  |
 --> ${DIRECTORY}/parsing/unterminated-pi-type-literal.lushui:3:26
  |
3 | data X (A: Type): (m: M) =
  |                          ^ unexpected token
  |

error: aborting due to previous error

